cupsfilters/pdftoraster.cxx

```diff
@@ -1446,6 +1446,8 @@ write_page_image(cups_raster_t *raster,
         if (doc->header.cupsBitsPerColor == 1) // Special case for 1-bit
 	                                       // colorspaces
 	{
+	  //pr.set_render_hint(poppler::page_renderer::antialiasing, false);
+	  pr.set_render_hint(poppler::page_renderer::text_antialiasing, false);
```

Comment:
What about set it to false at the beginning for 1bit colorspace, so the function is not called twice?

Reply:
Done, rebased and force-pushed.